File: framework/src/play-ws/src/main/scala/play/api/libs/ws/WSBodyReadables.scala
/*
 * Copyright (C) 2009-2017 Lightbend Inc. <https://www.lightbend.com>
 */
package play.api.libs.ws

/**
 * A set of useful [[BodyReadable]] implicit classes.
 */
trait WSBodyReadables extends DefaultBodyReadables with JsonBodyReadables with XMLBodyReadables

object WSBodyReadables extends WSBodyReadables

File: framework/src/play-ws/src/main/java/play/libs/ws/WSBodyReadables.java
/*
 * Copyright (C) 2009-2018 Lightbend Inc. <https://www.lightbend.com>
 */

package play.libs.ws;

/**
 * JSON, XML and Multipart Form Data Readables used for Play-WS bodies.
 */
public interface WSBodyReadables extends DefaultBodyReadables, JsonBodyReadables, XMLBodyReadables {
    WSBodyReadables instance = new WSBodyReadables() {};
}
